CanalTP/Chaos
# Feature rzo 395 caching

# Description

This PR allows storage of navitia responses in cache

## Issue (optional)

Issue link: rzo-395

## Pull Request type (optional)

This is a new feature

## How to test

Here are the following steps to test this pull request:

- update requirements ```pip install -r requirements.txt```
- response time for disruption / impact API should be less than without

Edit default_settings.py for your needs :
```
CACHE_CONFIGURATION = {
    'CACHE_TYPE': 
```

'simple', 'redis' if installed or nothing for no-cache
